src/main/java/org/prebid/server/floors/BasicPriceFloorAdjuster.java

```diff
@@ -34,6 +44,21 @@ private static ExtRequestBidadjustmentfactors extractBidadjustmentfactors(BidReq
         return ObjectUtil.getIfNotNull(extPrebid, ExtRequestPrebid::getBidadjustmentfactors);
     }
 
+    private static ImpMediaType chooseMediaType(ExtRequestBidadjustmentfactors extBidadjustmentfactors, String bidder) {
```

Comment:
What if the lowest mediatype value is 0.7 for banner, but Imp even don't have banner mediatype? We should work only with mediatypes presented in Imp